src/Plugin/PasswordConstraint/Error.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Drupal\password_enhancements\Plugin\PasswordConstraint;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\PluginBase;
+use Drupal\password_enhancements\Plugin\PasswordConstraintPluginInterface;
+
+/**
+ * Lower-case password constraint plugin.
+ *
+ * @PasswordConstraint(
+ *   id = "error",
+ *   name = @Translation("Error handler"),
+ *   description = @Translation("Plugin for error handling."),
+ *   unique = TRUE,
+ *   internal = TRUE,
+ * )
+ *
+ * @internal Used only for error handling, this plugin shouldn't be selectable.
```

Comment:
Yeah, let's get rid of this workaround (see my comment in the module file) because if someone does not read this comment s/he may use it in a way that s/he should not...

Reply:
I would rather not, this is the easiest way to handle the errors in place, and validate it at the same time.

Reply:
Thrown out.